docs/content/widgets/Grids.js

```diff
@@ -53,6 +53,7 @@ export const Grids = <cx>
             <div controller={PageController}>
                 <Grid records:bind='$page.records'
                       style={{height: '300px'}}
+                      onCreateIsRecordSelectable={(_, { store }) => record => record.id % 2}
```

Comment:
Let's not add this here. This page might be a first encounter with the grid for the user and it will look like a bug if some records are not selectable.